src/StateSmithTest/CodeFileWriter/Ex2.h

```diff
@@ -67,5 +68,6 @@ struct Ex2
     Ex2_Func current_state_exit_handler;
 };
 
+#endif // EX2_H
```

Comment:
This change will break behavior for a handful of users, but I think that's OK. 99% of people want standard include guards instead of pragma once. I'll help document migration steps for anyone effected.

A small number of users (I'm guessing) post process their generated state machine files to add code to their .h file. https://github.com/StateSmith/StateSmith-examples/blob/main/user-post-process/README.md

You can see that here in the Ex2.h file. The user code appends ` Ex2_event_id_to_string()` at the end of the file.

We could make another render config setting for C so that users can choose whether to use pragma once, but I think we should leave it as you have for now. While this is a slight breaking change for a few users, I believe the vast majority of users would prefer standard include guards over my initial lazy pragma once :) It's also very easy for users to adapt to this change.

To help the few effected users, I will:
* post to the discord group and ask if anyone would be affected by the change and then help them migrate.
* update the user post processing example (adapting to the change is pretty trivial).
* add a small breaking changes note in the changelog

Thoughts?